ci/pipeline-config-ocp.yaml

```diff
@@ -32,6 +32,7 @@ production:
         - v4.10
         - v4.11
         - v4.12
+        - v4.13
```

Comment:
```suggestion
```


Running a `skopeo list-tags docker://registry.redhat.io/openshift4/ose-operator-registry` it looks like the 4.13 tag is not present yet. It shouldn't be a big issue because we added 4.12 just a few weeks ago so I don't think it will break anything.